physics/GFS_rrtmg_pre.F90

```diff
@@ -854,7 +968,7 @@ subroutine GFS_rrtmg_pre_run (Model, Grid, Sfcprop, Statein,   & ! input
                          clouds,cldsa,mtopa,mbota, de_lgth)                !  --- outputs
 
 
-        elseif(Model%imp_physics == 15) then
+        elseif(Model%imp_physics == 6 .or. Model%imp_physics == 15) then
```

Comment:
Hi @mzhangw @climbfuji @ChunxiZhang-NOAA ,

Should this line:
`elseif(Model%imp_physics == 6 .or. Model%imp_physics == 15) then`
be
`elseif(Model%imp_physics == 15) then`
?

Thanks!

Bin

Reply:
`elseif(Model%imp_physics == 15) then` is correct.

We don't support WSM6 (== 6) anyway, but if at all it should call the new progcld6 (= old progcld5, see answer below).

Reply:
Hi @climbfuji 

In the support/HAFS branch, I saw a
`elseif(Model%imp_physics == 8) then`
section, but in the new master, there is no '8' here anymore. Instead I saw this Model%imp_physics == 6 in
`elseif(Model%imp_physics == 6 .or. Model%imp_physics == 15) then`
Is the imp_physics == 6 in master corresponds to the imp_physics == 8 in support/HAFS, or the imp_physics == 8 has been cleaned up from the master?

But, anyway, to be consistent with the master, we will need to separate out the imp_physics == 6 option and let it call the progcld6 instead of progcld5.

Thanks!

Bin

Reply:
I think you can safely ignore `== 6`. The `== 8` should be further down (or up?) to branch into `progcld6`.

Reply:
Changes were committed to separate out the imp_physics == 6 option and combine it with the imp_physics == 8 option, so that it calls the progcld6 instead of progcld5.